ldes-server-infra-postgres/postgres-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/postgres/batch/MemberItemReader.java

```diff
@@ -36,7 +36,7 @@ public JdbcPagingItemReader<FragmentationMember> refragmentEventStream(@Value("#
 				.dataSource(dataSource)
 				.rowMapper(new MemberRowMapper())
 				.queryProvider(refragmentQuery())
-				.parameterValues(Map.of("viewName", jobParameters.get("viewName")))
+				.parameterValues(Map.of("viewName", jobParameters.get("viewName"), "collectionName", jobParameters.get("collectionName")))
```

Comment:
I never really understood the purpose of creating a new map with the same entries as the provided jobParameters map